gamebook/option.py

```diff
@@ -1,20 +1,27 @@
 class Option(object):
-    def __init__(self, text, key, decision):
-        self.text = text
+    def __init__(self, text, key, nextscene):
+        self.text = text 
         self.key = key
-        self.decision = decision
+        self.nextscene = nextscene
 
     def show_text(self):
         return self.text
 
-    def get_decision(self):
-        return self.decision.get_decision_name()
+    def show_nextscene(self):
+        nextscene_list = 'next scenes:'
+        for i in self.nextscene:    
+            scene = i.get_scene_name()
+            rate = i.get_success_rate()
+
+            nextscene_list +=f"{scene}, {rate}\n"
```

Comment:
The textual stuff you are returning will give you problems later on when you add an `Output` class. Try to return a dict.

Reply:
Then show it manually for now (later on the `Output` class will handle it).

Reply:
Another option is just to remember that you will need to change this later on.

Reply:
ok i'll change it to dict

Reply:
@arielofer don't, later when you add the Output class we will refactor this.

Reply:
:+1: